src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Ball {

```

Comment:
```suggestion
public class Ball {
```

Reply:
we would usually make all of these top-level classes public

Reply:
ok, done